pcdsdevices/lens.py

```diff
@@ -41,6 +41,8 @@ class LensStack(PseudoPositioner):
 
     calib_z = Cpt(PseudoSingle)
 
+    tab_whitelist = ['x', 'y', 'z', 'calib_z', 'tweak', 'align']
```

Comment:
```suggestion
    tab_whitelist = ['tweak', 'align']
    tab_component_names = True
```

Reply:
For some reason I thought this would collect component names from parent classes, but it doesn't.

Reply:
Huh, ok...